docs/src/pages/demos/app-bar/app-bar.md

```diff
@@ -42,3 +42,46 @@ A side searchbar.
 ## Bottom App Bar
 
 {{"demo": "pages/demos/app-bar/BottomAppBar.js", "iframe": true, "maxWidth": 500}}
+
+## Scrolling
+
+### Hide App Bar
+
+An App Bar that hides on scroll.
+
+{{"demo": "pages/demos/app-bar/HideAppBar.js", "iframe": "true", "maxWidth": 500}}
+
+### Elevate App Bar
+
+An App Bar that elevates on scroll.
+
+{{"demo": "pages/demos/app-bar/ElevateAppBar.js", "iframe": "true", "maxWidth": 500}}
+
+### `useScrollTrigger([options]) => trigger`
+
+#### Arguments
+
+1. `options` (*Object* [optional]):
+
+  - `options.disableHysteresis` (*Boolan* [optional]): Defaults to `false`. Disable the hysteresis. Ignore the scroll direction to determine the `trigger` value.
+  - `options.target` (*Node* [optional]): Defaults to window.
```

Comment:
```suggestion
  - `options.target` (*Node* [optional]): Defaults to `window`.
```

Reply:
Currently, some "Defaults" statements don't use backticks but they probably all should.